### ISSUE TITLE
[BUG] PDF does not generate

### ISSUE THREAD

**Issue by philipdistefano** [AmruthPillai/Reactive-Resume#907]
**Describe the bug**

Creating a test resume and attempted export a PDF does not work.  Docker logs show the following:

```
 ERROR [ExceptionsHandler] page.goto: net::ERR_CONNECTION_REFUSED at http://192.168.0.243:3000/user/test2/printer?secretKey=secretkeyishere
=========================== logs ===========================
navigating to "http://192.168.0.243:3000/user/test2/printer?secretKey=secret key is here", waiting until "load"
```

**Product Flavor**
- [ ] Managed (https://rxresu.me)
- [X ] Self Hosted

**Expected behavior**
A PDF should export and be available to download.

**Additional context**
 - OS: Running in docker on Ubuntu 20.04
 - Version RxResume v3.4.5

**Comment by philipdistefano** [comment]
Updated to 3.4.6 and realized this wasn't a bug, but the port wasn't exposed.  Not sure what's wrong with my setup to where I have to expose the port, but once I added 3000:3000 and 3100:3100 under ports for client/server respectively, this issue progressed to the next error, which is:

```
 [Nest] 43  - 06/19/2022, 5:10:00 PM   ERROR [ExceptionsHandler] page.waitForSelector: Timeout 30000ms exceeded.
=========================== logs ===========================
waiting for selector "html.wf-active" to be visible
============================================================
page.waitForSelector: Timeout 30000ms exceeded.

```

**Comment by mpr-abond** [comment]
I'm running into this same issue.  Exact same errors (connection refused, then wait timeout).

**Comment by qiqian** [comment]
same html.wf-active timeout issue
this doesn't seem to be actively maintained
need to manually fix health check, figure out the traefix configs, and just get stuck with the pdf export

**Comment by AmruthPillai** [comment]
Fixed in [v3.6.0](https://github.com/AmruthPillai/Reactive-Resume/releases/tag/v3.6.0)
